packages/bitcore-wallet-service/src/serverMessages.ts

```diff
@@ -1,8 +1,8 @@
-module.exports = (wallet, appName, appVersion) => {
+module.exports = (wallet, appName, appVersion, userAgent) => {
   if (!appVersion || !appName) return;
 
   const serverMessages = [];
-  if (wallet.network == 'livenet' && appVersion.major == 5 && wallet.createdOn < 1443461026) {
+  if (wallet.network == 'livenet' && appVersion.major == 5) {
```

Comment:
we can remove this block now.

Reply:
This is used for testing purpose, remove this anyway?